src/libraries/Wasm.Build/tests/Directory.Build.props

```diff
@@ -0,0 +1,6 @@
+<Project>
```

Comment:
It _sounds_ like this should be "functional tests" instead in https://github.com/dotnet/runtime/tree/master/src/tests/FunctionalTests.

@MaximLipnin knows the details but I think the setup is basically the same as what you have here.

Reply:
This is adding a new scenario for running wasm related tests, but with xunit. I did this as a scenario, so we can setup the dependencies (eg. xunit runner, sdk etc), and have them in a specific directory so it's easy to pick them up for sending to helix.

Functional tests will have a different requirement for sdk etc.
IIUC, FunctionalTests use the `src/libraries/` setup - https://github.com/dotnet/runtime/blob/master/src/tests/FunctionalTests/Directory.Build.props#L10 .

Reply:
1. `RunOutsideWasm` => `RunTestsWithXUnit`?
2. Move the tests to `src/tests/RunWithXUnit/Wasm.Build.Tests` .. or something similar?